UPGRADE-2.0.md

```diff
@@ -4,3 +4,6 @@ UPGRADE FROM 1.x to 2.0
  * The `getServer()` method has been removed from the CommandFailedEvent,
    CommandStartedEvent, and CommandSucceededEvent event classes. The `getHost()`
    and `getPort()` methods have been added in its place.
+ * All classes that previously implemented the `Serializable` interface no
+   longer implement this. This changes the serialization format, but the 
+   ability to serialize and unserialize these classes remains as-is. 
```

Comment:
I don't think we need to say "this changes the serialization format" as the [Serializable](https://www.php.net/manual/en/class.serializable.php) interface hasn't been used in some time.

In PHP 7.4+, the `__serialize()` and `__unserialize()` magic methods are preferred when available (see: [docs](https://www.php.net/manual/en/language.oop5.magic.php#language.oop5.magic.serialize)). We also noted this in [PHPC-2230](https://jira.mongodb.org/browse/PHPC-2230?focusedCommentId=5505609&focusedId=5505609&page=com.atlassian.jira.plugin.system.issuetabpanels:comment-tabpanel#comment-5505609) when dropping support for PHP 7.2 and 7.3 in 1.17.0.

----

Presumably, the only loss in functionality here is that users would no longer be able to unserialize a string in the legacy "C format" since that would depend on `Serializable::unserialize()`. All of the tests for handling that format were removed in https://github.com/mongodb/mongo-php-driver/commit/2ab1e4ba2e6b3126e7acbd7025f3a2727aaa48d2.

Reply:
Good point. I removed the second sentence and only highlight that the interface is no longer implemented.